model/src/main/proto/spotlight.proto

```diff
@@ -5,55 +5,72 @@ package model;
 option java_package = "org.oppia.android.app.model";
 option java_multiple_files = true;
 
-// Superclass for all the SpotlightViewStates
+// Spotlights are visual elements that are shown during the initial user onboarding experience of
+// the app that highlight certain UI elements and show how/what they should be used for.
 message Spotlight {
   oneof feature {
```

Comment:
The oneof itself should also be documented since it also corresponds to generated code.

Reply:
done - please check if the doc looks fine